examples/mysql-wordpress-pd/wordpress-deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: wordpress
+  labels:
+    app: wordpress
```

Comment:
In general, I'd put the same labels on the Service and PVC as on the corresponding Deployment.